docs/dataset/advanced-usage/formatting-your-datasets.md

```diff
@@ -50,10 +50,14 @@ Kolena will look for the following fields when displaying datapoints:
 |------------|---------------------------------------------------------------------------------------------------------------------------------------------|
 | `locator`  | Url path to a file to be displayed, either a [cloud storage](../../connecting-cloud-storage/index.md) url or a http url that serves a file. |
 | `text`     | Raw text input for text based models.                                                                                                       |
+| `file_type`| Optional field indicating the type of file (e.g., `jpg`, `mp4`). Used to assist in the inference of the datatype.                           |
 
-A locator needs to have correct extensions for the corresponding file type. For example an image should be in a format
+The `locator` needs to have correct extensions for the corresponding file type. For example an image should be in a format
 such as `.jpg` or `.png`, whereas locators for audio data should be in forms like `.mp3` or `.wav`.
 
+The optional `file_type` field can be used to explicitly provide the file type information
+when extensions cannot be included in the `locator`.
+
```

Comment:
> Is there any risk of someone interpreting this as file_type of image (and putting in data type)? Should we handle that?

@munkyshi, i don't think we want to handle it for now, but just updated the docs to address the concern. does this make sense to you?

<img width="889" alt="Screen Shot 2024-06-17 at 2 42 13 PM" src="https://github.com/kolenaIO/kolena/assets/119692499/df519626-0841-4bcf-a35f-24dfec604266">

Reply:
Should we call it `file_extension` in that case?

Reply:
yeah, `Explicit is better than implicit.`.
just rename it in [this commit](https://github.com/kolenaIO/kolena/pull/620/commits/207b30cebd9d5a0451abd99308e95fa69644dae1)